go/libraries/doltcore/sqle/dtables/column_diff_table.go

```diff
@@ -0,0 +1,656 @@
+// Copyright 2022 Dolthub, Inc.
```

Comment:
I think this change got reset.

```suggestion
// Copyright 2023 Dolthub, Inc.
```